README.md

```diff
@@ -2,51 +2,32 @@
 
 [![ipfs](http://img.shields.io/irc/%23ipfs.png)](https://webchat.freenode.net/?channels=ipfs)
 
-IPFS is a distributed file system that seeks to connect all computing devices with the same system of files. In some ways, this is similar to the original aims of the Web, but IPFS is actually more similar to a single bittorrent swarm exchanging git objects.
+Welcome to IPFS (the InterPlanetary File System)! Why not [watch a video](https://www.youtube.com/watch?v=8CMxDNuuAiQ) to get started?
 
-IPFS could become a new major subsystem of the internet. If built right, it could complement or replace HTTP. It could complement or replace even more. It sounds crazy. It _is_ crazy.
-
-#### IPFS IS A WORK IN PROGRESS
-
-Please note that IPFS is a work in progress. It is an ambitious plan to make the internet more free, open, secure, and high performance. It builds on the good ideas of numerous battle-tested distributed systems.
-
-[![Ready](https://badge.waffle.io/ipfs/ipfs.png?label=ready&title=Backlog)](https://waffle.io/ipfs/ipfs)
-
-## Security Issues
-
-The IPFS protocol and its implementations are still in heavy development. This means that there may be problems in our protocols, or there may be mistakes in our implementations. And -- though IPFS is not production-ready yet -- many people are already running nodes in their machines. So we take security vulnerabilities very seriously. If you discover a security issue, please bring it to our attention right away!
+## Table of Contents (links)
 
-If you find a vulnerability that may affect live deployments -- for example, by exposing a remote execution exploit -- please send your report privately to security@ipfs.io. Please DO NOT file a public issue.
-
-If the issue is a protocol weakness that cannot be immediately exploited or something not yet deployed, just discuss it openly.
+- [Overview](#overview)
+- [How IPFS Works](#how-ipfs-works) ([Talks](#ipfs-talks)) ([Demos](#demos-and-examples))
```

Comment:
Maybe "What is IPFS (Talks, Papers, Demos, How it works)"

- the big list is a summary, not really an explanation.
- add "Papers" link too
- "how it works" could have a few paragraphs expanding on the opening paragraph.